### PR TITLE
LPS-119036 Image placeholder and pagination arrows are wrong in RTL languages

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hi guys,

Can you review this pull request, please?

There are two issue to resolve here when we change to RTL view:

1. Image placeholders are not centered.
2. Pagination arrows are pointing inwards instead of outwards.

You can find more information about how to reproduce the issue in [LPS-119036](https://issues.liferay.com/browse/LPS-119036). 

1. The solution for the first issue is just based on applying current styles to image placeholder but considering rtl languages. 
2. The solution for the second case is based in what we are already doing ([here](https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/page_iterator/start.jsp#L271)) even in the same taglib to consider these languages.

Thanks.

Regards.

/cc @NemethNorbert

### CODE REVIEW

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_vertical_card.scss [hunk 1]
```diff
@@ -49,3 +49,12 @@
...
 }
+
+.rtl {
+	.card .aspect-ratio-item-center-middle {
+		left: auto;
+		margin-right: 1.5rem;
+		right: 50%;
+		transform: translate(-100%, -50%);
+	}
+}
```
Comment (wincent) [inline modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_vertical_card.scss:60]: We pretty rarely have to explicitly target `.rtl` (I count 16 hits in all of liferay-portal). Any idea why the transform that our frontend-css-rtl-servlet does isn't covering us here? cc @eduardoallegrini

**Comment by wincent** [comment]
ci:test:sf

**Comment by wincent** [comment]
ci:test:relevant

**Comment by wincent** [comment]
ci:test:bundle

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_vertical_card.scss (hunk 1 above)
Comment (eduardoallegrini) [reply modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_vertical_card.scss:60]: The script is working but it's not changing the `translate` values

ezgif com-video-to-gif

So
```scss
.rtl {
	.card .aspect-ratio-item-center-middle {
		transform: translate(50%, -50%);
	}
}
```
Should be enough

**Comment by carloslancha** [comment]
Shouldn't the image icon be flipped too? The "text" in the icon looks like is written ltr.

**Comment by NemethNorbert** [comment]
Hey,

I am on PTO till monday, so sorry for not responding to the requested change. I just made the css change and pushed it.

**Comment by NemethNorbert** [comment]
ci:test:sf

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Review by eduardoallegrini — APPROVED** [review]

**Comment by jbalsas** [comment]
> Shouldn't the image icon be flipped too? The "text" in the icon looks like is written ltr.

That sounds reasonable, but maybe not worth the trouble. We've done some of that for the editor, but we don't necessarily do that anywhere else, so this would need to be a coordinated general effort for it to be consistently applied.

Since @eduardoallegrini approved and is green, going to push it forward, we can always continue improving it!

**Comment by jbalsas** [comment]
ci:forward

**Pull request closed without merging**